setup.cfg

```diff
@@ -50,6 +50,8 @@ filterwarnings =
     error:::pandas[.*]
     error:::numpy[.*]
     error:::distributed[.*]
+    # From https://github.com/numpy/numpy/issues/20225
+    ignore:The distutils.sysconfig module is deprecated, use sysconfig instead:DeprecationWarning
```

Comment:
NumPy is in the process of [deprecating distutils]( https://numpy.org/devdocs/reference/distutils_status_migration.html ), but that will probably take a few releases. Given as we do like to test these older releases, we should probably leave this for a while and revisit in the future